File: cocos2dx-lua/poco/sdk/Attributor.lua
local Attributor = {}
Attributor.__index = Attributor

function Attributor:getAttr(node, attrName)
    local node_ = node
    if node.__isPocoNodeWrapper__ == nil then
        node_ = node[1]
    end
    return node_:getAttr(attrName)
end

function Attributor:setAttr(node, attrName, attrVal)
    local node_ = node
    if node.__isPocoNodeWrapper__ == nil then
        node_ = node[1]
    end
    node_:setAttr(attrName, attrVal)
end

return Attributor

File: cocos-creator/3X/sdk/Attributor.ts
export namespace Attributor{
    function getAttr(node, attrName) {
        let node_ = node
        if (!node.__isPocoNodeWrapper__) {
            node_ = node[0]
        }
        return node_.getAttr(attrName)
    }
    function setAttr (node, attrName, attrVal) {
        let node_ = node
        if (!node.__isPocoNodeWrapper__) {
            node_ = node[0]
        }
        node_.setAttr(attrName, attrVal)
    }
}